packages/reward-root-submitter/README.md

```diff
@@ -79,3 +83,13 @@ You can release a new version on staging by running the following command in the
      waypoint up -app=reward-submit -prune-retain=0
 
 This will remove the previous deployment, build the docker image and deploy a new service.
+
+## Issues running on M1
+
+If you use an m1 (arm64 architecture) you may encounter errors when sub-dependencies do not have arm64 distribution.
+
+One way to resolve this is to follow [here](https://towardsdatascience.com/how-to-use-manage-multiple-python-versions-on-an-apple-silicon-m1-mac-d69ee6ed0250). The solution is to use Rosetta for `x86` terminal emulation. After doing so, you will have a separate pdm installation in `/usr/local/bin` 
+
```

Comment:
Important to ensure that your emulated pdm is pointing to the correct python -- tripped me up quite a bit.